en/Plugins/Releasing/Plugin guidelines.md

```diff
@@ -376,3 +376,28 @@ async function AsyncTest(): Promise<string | null> {
   }
 }
 ```
+
+## Language
+
+### English-first
+
+Write the following in English by default:
+
+- `id` property in `manifest.json`
+- `name` property in `manifest.json`
+- `description` property in `manifest.json`
+- `README.md`
+
+You may append translations to each property (except `id`) and document, but the English version must appear first.
```

Comment:
I'm not sure if mixing languages in the `description` is a good idea (or is that what you are referring to with `document`?